README.md

```diff
@@ -7,27 +7,34 @@ Kafka-based storage for Zipkin.
 > This is in experimentation phase at the moment.
 
 ```
-                    +-----------------------------zipkin------------------------------------------
-                    |                                                         +-->( service:span )
-                    |                                                         +-->( span-index   )
-( collected-spans )-|->[ span-consumer ]                      [ span-store ]--+-->( traces       )
-                    |         |                                    ^    |     +-->( dependencies )
-                    +---------|------------------------------------|----|--------------------------
-                              |                                    |    |
-------------------------------|------------------------------------|----|--------------------
-                              |                                    |    |
-kafka                         +-->(  raw-spans  )--// potential //-+    +-->( traces )
-                              |                    // enriching // |    |
-                              +-->( light-spans )--// sampling  //-+    +-->( dependencies )
-
---------------------------------------------------------------------------------------------
+                    +----------------------------*zipkin*-------------------------------------------
+                    |                                                           +-->( service:span )
+                    |                                                           +-->( span-index   )
+( collected-spans )-|->[ span-consumer ]        [ aggregation ] [ span-store ]--+-->( traces       )
```

Comment:
more seriously than last comment.. heh, there are two major class of filter/mutators span and trace based. the former being stateless so can apply before your aggregations and possibly lower the total size. will start on that today based on this thread https://github.com/openzipkin/zipkin/pull/2072

Reply:
Awesome!! 

I found the combination of Kafka Streams [Session Windows](https://kafka.apache.org/20/documentation/streams/developer-guide/dsl-api.html#session-windows) and [Supression](https://www.confluent.io/blog/kafka-streams-take-on-watermarks-and-triggers) as a great fit to enable trace-based filters. e.g. It helped to finally get accurate dependency numbers.